### DISCUSSION TITLE
Getting Server Error 500 when trying to post to url received in edc callback

### DISCUSSION THREAD

**Discussion by anandt1201** [eclipse-tractusx/tractusx-edc#590] [Q&A]
Hi, 
I am getting Server Error 500 when trying to post to url received in edc callback with key as Authorization and token received in callback.
![image](https://github.com/eclipse-tractusx/tractusx-edc/assets/30823144/863be8dc-7080-4f14-8c6b-0e390a342e2e)

This is the log that I receive in dataplane
![image](https://github.com/eclipse-tractusx/tractusx-edc/assets/30823144/984ca05b-644e-49ab-9599-fd95d7a80e0b)

This is the log of control plane
![image](https://github.com/eclipse-tractusx/tractusx-edc/assets/30823144/81884ccb-9e33-4b06-aa1c-26fa75a3a4b7)

**Comment by stephanbcbauer** [comment]
@anandt06 What did you do before? Did you update the edc? Which version are you using? Which environment? THX

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
